server/netty-server/loom/src/main/scala/sttp/tapir/server/netty/loom/loom.scala

```diff
@@ -0,0 +1,20 @@
+package sttp.tapir.server.netty
+
+import sttp.monad.MonadError
+
+package object loom {
```

Comment:
maybe this should be an `id` package?

Reply:
or ... maybe we can put the whole `Id` interpreter into the main package? only problem, we'd need to compile it using JDK 21. But maybe we can compile using JDK 21 and target 11 bytecode?

Then we could also share the `Id` alias across netty-loom / nima

Reply:
but maybe it's better to keep this separate, I don't know ;)

Reply:
yeah let's not overcomplicate for now ;)